src/network/client.rs

```diff
@@ -42,6 +45,23 @@ impl DHTCell {
 	}
 }
 
+struct DHTRow((RowIndex, Vec<u8>));
+
+impl DHTRow {
+	fn reference(&self, block: u32) -> String {
+		self.0 .0.reference(block)
```

Comment:
Is this a unique ref per block?

Reply:
Row reference is block_number:row_number, it should be unique. Once we have proper type, I guess monstrous constructions like `self.0.0` will disappear.